templates/index.html

```diff
@@ -29,7 +32,7 @@
          }
         </style>
     </head>
-    <body>
+    <body id="page">
```

Comment:
why is this here

Reply:
oh yep you're, this slipped in, let me remove it